packages/pangea-sdk/pangea/asyncio/services/file_scan.py

```diff
@@ -111,7 +111,7 @@ async def file_scan(
             source_url=source_url,
         )
         data = input.dict(exclude_none=True)
-        return await self.request.post("v1/scan", m.FileScanResult, data=data, files=files, poll_result=sync_call)
+        return await self.request.post("v1/scan", m.FileScanResult, data=data, files=files, poll_result=sync_call)  # type: ignore[arg-type]
```

Comment:
An alternative to this would be to either:

1. Use the covariant `Sequence` instead of `List`:

```diff
diff --git a/packages/pangea-sdk/pangea/asyncio/request.py b/packages/pangea-sdk/pangea/asyncio/request.py
index e3775bd..44a510b 100644
--- a/packages/pangea-sdk/pangea/asyncio/request.py
+++ b/packages/pangea-sdk/pangea/asyncio/request.py
@@ -5,7 +5,7 @@ import asyncio
 import json
 import os
 import time
-from typing import Dict, List, Optional, Tuple, Type, Union
+from typing import Dict, List, Optional, Sequence, Tuple, Type, Union
 
 import aiohttp
 from aiohttp import FormData
@@ -30,7 +30,7 @@ class PangeaRequestAsync(PangeaRequestBase):
         endpoint: str,
         result_class: Type[PangeaResponseResult],
         data: Union[str, Dict] = {},
-        files: Optional[List[Tuple]] = None,
+        files: Optional[Sequence[Tuple]] = None,
         poll_result: bool = True,
         url: Optional[str] = None,
     ) -> PangeaResponse:
@@ -147,7 +147,7 @@ class PangeaRequestAsync(PangeaRequestBase):
 
         return await self.poll_result_by_id(request_id, response.result_class, check_response=check_response)
 
-    async def post_presigned_url(self, url: str, data: Dict, files: List[Tuple]):
+    async def post_presigned_url(self, url: str, data: Dict, files: Sequence[Tuple]):
         # Send form request with file and upload_details as body
         resp = await self._http_post(url=url, data=data, files=files, presigned_url_post=True)
         self.logger.debug(
@@ -262,7 +262,7 @@ class PangeaRequestAsync(PangeaRequestBase):
         url: str,
         headers: Dict = {},
         data: Union[str, Dict] = {},
-        files: Optional[List[Tuple]] = None,
+        files: Optional[Sequence[Tuple]] = None,
         presigned_url_post: bool = False,
     ) -> aiohttp.ClientResponse:
         if files:
@@ -303,7 +303,7 @@ class PangeaRequestAsync(PangeaRequestBase):
         endpoint: str,
         result_class: Type[PangeaResponseResult],
         data: Union[str, Dict] = {},
-        files: Optional[List[Tuple]] = None,
+        files: Optional[Sequence[Tuple]] = None,
     ):
         if files is None or len(files) == 0:
             raise AttributeError("files attribute should have at least 1 file")
diff --git a/packages/pangea-sdk/pangea/asyncio/services/file_scan.py b/packages/pangea-sdk/pangea/asyncio/services/file_scan.py
index f6bd22a..d5de5f4 100644
--- a/packages/pangea-sdk/pangea/asyncio/services/file_scan.py
+++ b/packages/pangea-sdk/pangea/asyncio/services/file_scan.py
@@ -111,7 +111,7 @@ class FileScanAsync(ServiceBaseAsync):
             source_url=source_url,
         )
         data = input.dict(exclude_none=True)
-        return await self.request.post("v1/scan", m.FileScanResult, data=data, files=files, poll_result=sync_call)  # type: ignore[arg-type]
+        return await self.request.post("v1/scan", m.FileScanResult, data=data, files=files, poll_result=sync_call)
 
     async def request_upload_url(
         self,
```

2. Give the `files` variable above a type annotation of `List[Tuple]`.

Reply:
Fixed